index.html

```diff
@@ -0,0 +1,202 @@
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+        rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+            rel="stylesheet">
+</head>
+
+<body>
+    
+    <!----------navbar setion------------------->
+    <header>
+    <nav class="nav-container">
+        <a href="">John Doe</a>
+        <ul class="items-menu">
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+            <li><i class="fa-solid fa-envelope"></i></li>
+        </ul>
+        <a href="#"><i class="fa-solid fa-bars"></i></a>
+    </nav>
+    </header>
+    
+    <!--------------Hero section------------------------------------------->
+    <section>
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <ul>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+                </ul>
+            </div>
+            <div class="hero-right">
+                <h1>Hi There.<br>I am Hyder</h1>
+                <h5>I'm a web Developer</h5>
+                <p>I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you
+                    see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div class="hero-right1">
+                
+            </div>
+        </div>
+
+    </div>
+    </section>
+    <!------------------myworks---------------------------------------->
+    <div class="myworks">
+        <div class="work-heading">
+            <h4>My Recent Works</h4>
+            <hr class="hline">
+        </div>
+        <div class="work-details">
+            <div class="work-details-row1">
+                <img src="./images/Img Placeholder (1).png">
+                <div class="details-right1">
+                    <h4>Multi-Post Stories</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has
+                        been the industry's
+                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <span>HTML</span><span>CSS</span><span>Javascript</span>
+                    <p class="btn">See Projects</p>
```

Comment:
- [ ] This element looks good :+1: :100:, but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) it should be a `<button>` or `<a>` tag instead `<p>`. 

- [ ] Also, don't forget that you should add `:hover` and `:active` interactions to each button inside your project because it will improve the user experience :+1: 

![Screenshot from 2023-01-17 05-07-39](https://user-images.githubusercontent.com/105117832/212869823-264af09a-efe9-4580-b2df-d1cc68f81bf3.png)